pkg/kotsadm/objects/kotsadm_objects.go

```diff
@@ -304,16 +301,7 @@ func KotsadmDeployment(deployOptions types.DeployOptions) (*appsv1.Deployment, e
 			Value: "http://localhost:8800",
 		},
 	}
-	if strings.HasPrefix(deployOptions.StorageBaseURI, "docker://") {
-		env = append(env, corev1.EnvVar{
-			Name:  "STORAGE_BASEURI",
-			Value: deployOptions.StorageBaseURI,
-		})
-		env = append(env, corev1.EnvVar{
-			Name:  "STORAGE_BASEURI_PLAINHTTP",
-			Value: strconv.FormatBool(deployOptions.StorageBaseURIPlainHTTP),
-		})
-	} else {
+	if deployOptions.IncludeMinio {
```

Comment:
kotsadm deployment should always have an object store, let's keep the old logic where this check wasn't there? because it will fail differently

Reply:
🤦 right, the deployment will always need one
it's the statefulset that won't

I'll fix